site/documentation/content/api/command-and-control-kafka/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Command & Control API for Kafka Specification"
+linkTitle: "Command & Control API for Kafka"
+weight: 419
+---
+
+The *Command & Control* API of Eclipse Hono&trade; is used by *Business Applications* to send commands to connected devices.
+
+Commands can be used to trigger actions on devices. Examples include updating a configuration property, installing a software component or switching the state of an actuator.
+<!--more-->
+
+Hono distinguishes two types of commands. The first type is *one-way* only. In this case the sender of the command does not expect the device to send back a message in response to the command. This type of command is referred to as a *one-way command* in the remainder of this page. One-way commands may be used to e.g. *notify* a device about a change of state.
+
+The second type of commands expects a *response* to be sent back from the device as a result of processing the command. In this case the response contains a *status* code which indicates whether the command could be processed successfully. If so, the response may also include data representing the result of processing the command. This type of command is plainly referred to as a *command* because it represents the default case.
+
+The Command & Control API for Kafka is an alternative to the [Command & Control API for AMQP]({{< relref "/api/command-and-control" >}}).
+With this API, clients send command messages to an Apache Kafka&reg; cluster instead of an AMQP Messaging Network.
+
+{{% note title="Tech preview" %}}
+The support of Kafka as a messaging system is currently a preview and not yet ready for production. The APIs may change with the next version.
```

Comment:
In the specification of the Kafka-based downstream APIs the phrasing of the last sentence is now "The APIs are subject to change without prior notice". You may want to use it here as well.

Reply:
I've changed it accordingly.